src/services/api.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+const API_KEY = '16da3c7c6882c8c6fe2e82b63b8f8823';
```

Comment:
Never push API keys or secret keys to Github, take a look at `dotenv`.
It's not a big deal here, as it's a challenge using a public API, but in a real project, it would be a serious issue.

Reply:
Thanks for pointing that out. I had that in mind when i was coding, and only pushed it because it was not that a big deal as you said.